scripts/superdesk-dashboard/module.js

```diff
@@ -1,13 +1,21 @@
 (function() {
     'use strict';
 
-    DashboardController.$inject = ['$scope', 'desks', 'dashboardWidgets', 'api', 'session', 'workspaces', 'modal', 'gettext'];
-    function DashboardController($scope, desks, dashboardWidgets, api, session, workspaces, modal, gettext) {
+    DashboardController.$inject = ['$scope', 'desks', 'dashboardWidgets', 'api', 'session', 'workspaces', 'modal', 'gettext', 'privileges'];
+    function DashboardController($scope, desks, dashboardWidgets, api, session, workspaces, modal, gettext, privileges) {
         var vm = this;
 
         $scope.edited = null;
         $scope.workspaces = workspaces;
         $scope.$watch('workspaces.active', setupWorkspace);
+
+        // canConfigure returns true if the current user is allowed to configure
+        // the active workspace. The active workspace may be configured if it is
+        // a custom workspace or if the user has desk management privileges.
+        $scope.canConfigure = function() {
```

Comment:
imo this could be set in `setupWorkspace`, it won't change, so instead of function it could be just a value